great_expectations/render/renderer/column_section_renderer.py

```diff
@@ -394,40 +392,6 @@ def _render_value_counts_bar_chart(cls, evrs):
     def _render_failed(cls, evrs):
         return ExceptionListContentBlockRenderer.render(evrs, include_column_name=False)
 
-    @classmethod
-    def _render_unrecognized(cls, evrs, content_blocks) -> None:
-        unrendered_blocks = []
-        new_block = None
-        for evr in evrs:
-            if evr.expectation_config.expectation_type not in [
-                "expect_column_to_exist",
-                "expect_column_values_to_be_of_type",
-                "expect_column_values_to_be_in_set",
-                "expect_column_unique_value_count_to_be_between",
-                "expect_column_proportion_of_unique_values_to_be_between",
-                "expect_column_values_to_not_be_null",
-                "expect_column_max_to_be_between",
-                "expect_column_mean_to_be_between",
-                "expect_column_min_to_be_between",
-            ]:
-                new_block = TextContent(**{"content_block_type": "text", "text": []})
-                new_block["content"].append(
```

Comment:
For context, we determined this was dead code.

1. `TextContent` doesn't implement`__getitem__ `, so this would break.
2. `_render_unrecognized` is never called anywhere in the codebase.